### PR TITLE
Add `repeated-keyword` example

### PR DESCRIPTION
**Pull request by harupy**
## Type of Changes

|     | Type                   |
| --- | ---------------------- |
| ✓   | :scroll: Docs          |

## Description

Refs #5953

Closes #XXXX

### CODE REVIEW

File: doc/data/messages/r/repeated-keyword/bad.py [hunk 1]
```diff
@@ -0,0 +1,6 @@
+def func(a, b, c):
+    return a, b, c
+
+
+func(1, 2, c=3, **{"c": 4})  # [repeated-keyword]
+func(1, 2, **{"c": 3}, **{"c": 4})  # [repeated-keyword]
```
Comment (Pierre-Sassoulas) [inline doc/data/messages/r/repeated-keyword/bad.py:6]:
```suggestion
print("Hello ", "orld", sep="W", end=" !", **{"end": " :("})  # [repeated-keyword]
print("Hello ", "orld", **{"sep": "W", "end": " !"}, **{"end": " :("})  # [repeated-keyword]
```

File: doc/data/messages/r/repeated-keyword/good.py [hunk 2]
```diff
@@ -1 +1,5 @@
-# This is a placeholder for correct code for this message.
+def func(a, b, c):
+    return a, b, c
+
+
+func(1, 2, c=3)
```
Comment (Pierre-Sassoulas) [inline doc/data/messages/r/repeated-keyword/good.py:5]:
```suggestion
print("Hello ", "orld", sep="W", end=" !")
print("Hello ", "orld", **{"sep": "W", "end": " !"})
```

**Review by Pierre-Sassoulas — COMMENTED** [review]
Let me know what you think :)

File: doc/data/messages/r/repeated-keyword/bad.py [hunk 3]
```diff
@@ -0,0 +1,2 @@
+print("Hello ", "orld", sep="W", end=" !", **{"end": " :("})  # [repeated-keyword]
+print("Hello ", "orld", **{"sep": "W", "end": " !"}, **{"end": " :("})  # [repeated-keyword]
```
Comment (harupy) [inline doc/data/messages/r/repeated-keyword/bad.py:2]: Interesting. These two lines don't raise an error.

Comment (harupy) [reply doc/data/messages/r/repeated-keyword/bad.py:2]:
```python
def print(*args, **kwargs):
    pass

print("Hello ", "orld", sep="W", end=" !", **{"end": " :("})  # [repeated-keyword]
print("Hello ", "orld", **{"sep": "W", "end": " !"}, **{"end": " :("}) # [repeated-keyword]
```

worked.

Comment (harupy) [reply doc/data/messages/r/repeated-keyword/bad.py:2]:
```python
def f(*args, **kwargs):
    pass

f("Hello ", "orld", sep="W", end=" !", **{"end": " :("})  # [repeated-keyword]
f("Hello ", "orld", **{"sep": "W", "end": " !"}, **{"end": " :("}) # [repeated-keyword]
```

also worked.

Comment (Pierre-Sassoulas) [reply doc/data/messages/r/repeated-keyword/bad.py:2]: Hurgh, look like a bug in pylint. Maybe we can't determine the signature of print properly.

Comment (Pierre-Sassoulas) [reply doc/data/messages/r/repeated-keyword/bad.py:2]: I opened #6856, but maybe there's another well known function that exists without the bug ?

Comment (harupy) [reply doc/data/messages/r/repeated-keyword/bad.py:2]: Thanks! How about `open`?

Comment (Pierre-Sassoulas) [reply doc/data/messages/r/repeated-keyword/bad.py:2]: Well this is not raising anything either:
```python
with open("file", "r", encoding="utf8", **{"encoding": "ascii"}) as f:
    f.read()
```
Let's keep your original example.

**Review by Pierre-Sassoulas — APPROVED** [review]

**Pull request merged**